.github/workflows/lint.yml

```diff
@@ -26,9 +26,3 @@ jobs:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/install
       - run: pnpm run lint
-
-  yamllint:
```

Comment:
we have fewer yaml files now, no need to keep it

Reply:
Should we not use our share linting setup:
https://github.com/lifinance/lifi-backend/blob/develop/.github/workflows/lint.yml

Reply:
Using shared one makes more sense

Reply:
updated @maxklenk